FAQ.md

```diff
@@ -27,4 +27,4 @@ Also potentially relevant: inference with Mask R-CNN on high-resolution images m
 
 #### Q: How do I use Detectron to train a model on a custom dataset?
 
-**A:** If possible, we strongly recommend that you first convert the custom dataset annotation format to the [COCO API json format](http://cocodataset.org/#download). Then, add your dataset to the [dataset catalog](lib/datasets/dataset_catalog.py) so that Detectron can use it for training and inference. If your dataset cannot be converted to the COCO API json format, then it's likely that more significant code modifications will be required. If the dataset you're adding is popular, please consider making the converted annotations publicly available; If code modifications are required, please consider submitting a pull request.
+**A:** If possible, we strongly recommend that you first convert the custom dataset annotation format to the [COCO API json format](http://cocodataset.org/#download). Then, add your dataset to the [dataset catalog](https://github.com/facebookresearch/Detectron/blob/master/detectron/datasets/dataset_catalog.py) so that Detectron can use it for training and inference. If your dataset cannot be converted to the COCO API json format, then it's likely that more significant code modifications will be required. If the dataset you're adding is popular, please consider making the converted annotations publicly available; If code modifications are required, please consider submitting a pull request.
```

Comment:
I'm not sure if we need to update this. The FAQ.md is mostly out of date as I didn't really get to updating it for DetectAndTrack. Might need a full re-write to be useful. For now maybe remove this change and we can merge the fix to  the README.